pkgs/desktops/gnome/core/gnome-control-center/default.nix

```diff
@@ -155,9 +155,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeCheckInputs = [
     dbus
+    python3.pkgs.pygobject3 # for test-networkmanager-service.py
     python3.pkgs.python-dbusmock
     setxkbmap
-    xvfb-run
+    xorgserver
```

Comment:
Add comment it is for Xvfb?

Interestingly, it has been used [since 3.29](https://gitlab.gnome.org/GNOME/gnome-control-center/-/commit/8c9be792f25ce4d636e55d0872944936e45517a4) (PR for dbusmock inclusion pending).